protos/bottle/core/v1/bottle.proto

```diff
@@ -4,10 +4,15 @@ package bottle.core.v1;
 
 import "bottle/fulfillment/v1/order.proto";
 import "bottle/fulfillment/v1/verification.proto";
-import "bottle/notification/order/v1/assembly_failure.proto";
-import "bottle/notification/user/v1/created.proto";
-import "bottle/notification/user/v1/password_changed.proto";
-import "bottle/notification/user/v1/password_reset.proto";
+
+import "bottle/account/events/v1/organization_created.proto";
```

Comment:
This seems to remove the assembly failure one. Not sure if that is intended or not. Also, if we are redoing this, we should probably change it to order failure considering the errors we are sending are before it even gets to production.